packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -73,7 +75,7 @@ def format(self, record):
 
 
 # SEE https://docs.python.org/3/library/logging.html#logrecord-attributes
-DEFAULT_FORMATTING = "log_level=%(levelname)s | log_timestamp=%(asctime)s | log_source=%(name)s:%(funcName)s(%(lineno)d) | log_msg=%(message)s"
+DEFAULT_FORMATTING = "log_level=%(levelname)s | log_timestamp=%(asctime)s | log_source=%(name)s:%(funcName)s(%(lineno)d) | log_uid=%(log_uid)s | log_msg=%(message)s"
```

Comment:
I am wondering why are we having the `log_` prefix in front of everything? it is clear to me that this is a log message already. Is there a reason for having it here?

Reply:
Just wanted to have some prefixes, for our "custom" logs + I think is then nice in the Graylog when you want to filter by them they are all together in the search, also when you define Grok pattern somehow its more standardized with the prefix (but that's just my personal opinion)